bin/_mocha

```diff
@@ -323,9 +323,25 @@ var args = program.args;
 if (!args.length) args.push('test');
 
 args.forEach(function(arg){
-  files = files.concat(utils.lookupFiles(arg, extensions, program.recursive));
+  var newFiles;
+  try {
+    newFiles = utils.lookupFiles(arg, extensions, program.recursive);
+  } catch(err) {
+    if (err.message.indexOf('cannot resolve path') === 0) {
+      console.error('Warning: Could not find any test files matching pattern: ' + arg);
+      return;
+    } else {
```

Comment:
Isn't it good practice to re-throw any other error that might not be this specific one? Otherwise it will just be swallowed